crates/biome_js_analyze/tests/specs/correctness/noGlobalEval/invalid.js

```diff
@@ -0,0 +1,7 @@
+eval("var a = 0");
```

Comment:
Could you add more tests?
Basically it would be better to pass all ESLint test cases.
ref: https://github.com/eslint/eslint/blob/3a22236f8d10af8a5bcafe56092651d3d681c99d/tests/lib/rules/no-eval.js#L22-L190

Reply:
@nissy-dev 
Ah, sorry. I will the test cases.
And upon further review, I realized that I overlooked the CONTRIBUTING.md nested in /crates/biome_analyze. The changes I made were all manual, which might differ from those typically made by 'just new-lintrule' through automatic changes. I will also review this point to ensure everything aligns properly.

Thanks for your comment.